config/prow/config_knative.yaml

```diff
@@ -965,3 +975,17 @@ periodics:
     - auto-release: true
       dot-dev: true
       go113: true
+
+  knative-sandbox/operator:
+    - continuous: true
+      dot-dev: true
+      go113: true
+    - nightly: true
+      dot-dev: true
+      go113: true
+    - dot-release: true
```

Comment:
We need a hack to set the org name as env var for release jobs for non-knative projects, see https://github.com/knative/test-infra/blob/master/config/prow/config_knative.yaml#L906. 
So please add below lines to these two job settings:
```
env-vars:
    - ORG_NAME=knative-sandbox
```

Reply:
So this is only for `-release` steps, correct?

Updated.